_pages/en_US/dsiware-backups.md

```diff
@@ -8,47 +8,32 @@ redirect_from:
   - installing-dsiware
 ---
 
-{% include toc title="Instructions" %}
+{% include toc title="DSiWare Backups" %}
 
 ## Requirements
-- The latest release of [maketmd](https://github.com/Tuxality/maketmd/releases/latest) (if you're trying to convert a .app to a .tmd)
 - The latest version of [GodMode9i](https://github.com/RocketRobz/godmode9i/releases/latest)
+  - Download the archive, extract the contents and place `GodMode9i.nds` anywhere on your SD card
 - A Nintendo DSi console with [Unlaunch](/unlaunch) installed
 
-## Instructions
+## Nintendo DSi - Instructions
 
-### Identifying the path to the .app file
+### Section I - Identifying the path to the .app file
 1. Boot your Nintendo DSi console by holding **(A) & (B)**
-2. Identify your application in the list
+   - This should take you to the Unlaunch Management Menu
+1. Identify your application in the list
    - A light green entry means that they are installed to NAND
 
 When you hover over the entry, the bottom screen should show the path. Take note of the path.
 
 An example path would be `nand:/title/0030005/484e4a45/content/00000003.app`
 {: .notice--info}
 
-### Extracting the file
+### Section II - Extracting the file
 
-1. Launch GodMode9i.
-2. Navigate to SYSNAND, then navigate through the path you took note of above
-3. Highlight the file, then hit Y to copy
-   - Which file you use depends on your use case. For hiyaCFW, use the `.tmd`. Otherwise, use the `.app`
-4. Navigate your SD card to where you'd like to place the file
-   - For hiyaCFW usage, it'll be in `/title/00030004`
-5. Hit Y to paste the file
+1. Launch GodMode9i from the Unlaunch Menu

```

Comment:
In a few places like here you use "Unlaunch Menu", I think that'd be a fine name too, but one consistent name (menu, file menu, management menu) should be stuck to